lib/pages/people/view/people_page.dart

```diff
@@ -27,26 +33,47 @@ class _PeoplePageState extends State<PeoplePage> {
   @override
   Widget build(BuildContext context) {
     return AppScaffold(
+      actions: [
+        AppScaffoldAction(
+          icon: Icons.search,
+          onPressed: () {
+            setState(() {
+              searchClosed = !searchClosed;
+              visibilitySearchBar = !visibilitySearchBar;
+            });
+          },
+        )
+      ],
       title: Text(S.of(context).navigationPeople),
       body: Container(
         child: FutureBuilder(
             future: people,
             builder: (_, snapshot) {
               if (snapshot.connectionState == ConnectionState.done) {
-                final List<Person> peopleData = snapshot.data;
-                return ListView.builder(
-                    itemCount: peopleData.length,
-                    itemBuilder: (_, index) {
-                      return ListTile(
-                        leading: CircleAvatar(
-                          backgroundImage:
-                              NetworkImage(peopleData[index].photo),
-                        ),
-                        title: Text(peopleData[index].name),
-                        subtitle: Text(peopleData[index].email),
-                        onTap: () => showPersonInfo(peopleData[index]),
-                      );
-                    });
+                peopleData = snapshot.data;
+                return ListView(
```

Comment:
Replace this with a `Column` and make the `PeopleList` scrollable (you'll need to wrap it with `Expanded` here, for it to work) so that the search bar stays at the top even if you scroll through the list.